data/distrodefs/fedora/imagetypes.yaml

```diff
@@ -2277,6 +2277,7 @@ image_types:
       - "distro"
       - "customizations.fips"
       - "customizations.locale"
+      - "customizations.installer"
```

Comment:
AIUI we support more than those [0] and we also need to update the other distros (rhel-*)?

[0] If not, having kernel.append would be very nice as it allows us to set console=ttyS0 so that we see what the installer is doing during the automatic test

Reply:
I did a quick experiment in https://github.com/osbuild/images/commit/144d6bcda88c77d9bc2ba222cbcfef687f15d362 (branch https://github.com/osbuild/images/compare/main...mvo5:qemu-runner-bib-netinst-real-test?expand=1)

Reply:
<img width="1771" height="640" alt="image" src="https://github.com/user-attachments/assets/ccdcf51b-885f-413b-99c5-a137ae5161ae" />

I'm getting the same and I think we explicitly need to set the user & group customisations in the yaml file otherwise it doesn't look like the get set. But ya, same issue with the source here

ETA: I built a fedora-42 iso (aarch) for what it's worth